Misc/NEWS.d/next/Tests/2018-09-09-14-36-59.bpo-34569.okj1Xh.rst

```diff
@@ -0,0 +1,3 @@
+fix pystate.c/ _new_long_object() so that a 32-bit signed object is returned
```

Comment:
Please start sentences with upper letter and add punctuation.

Reply:
After 12 hours of debugging and testing, sorry I did not notice that I missed he shift key. I'll post an updated PR when I have time.